Tests/TensorFlowTests/OptimizersTests.swift

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+@testable import TensorFlow
+
+final class OptimizerTests: XCTestCase {
+
+    func testAdaGrad() {
+        struct Classifier: Layer {
+            var l1, l2: Dense<Float>
+            init(hiddenSize: Int) {
+                l1 = Dense<Float>(
+                        inputSize: 2,
```

Comment:
```suggestion
                    inputSize: 2,
```

Indent all line breaks by 4 whitespaces. Same for other lines.